### DISCUSSION TITLE
Generate custom designs

### DISCUSSION THREAD

**Discussion by farhan2077** [resend/react-email#930] [Q&A]
Is it possible to make designs with react-email like this image below?
<img width="237" alt="image" src="https://github.com/resendlabs/react-email/assets/32035481/050f2307-b253-4258-a1ee-510aee54fc2e">

**Comment by jinsley8** [comment]
Yes, but background images aren't fully compatible across all email clients.

Email clients do not render the same as browsers so keep this in mind with email designs.

https://www.caniemail.com/features/css-background-image/

**Comment by dsamarin** [comment]
If you want full compatibility, I would recommend either a design like this:

![image](https://github.com/resendlabs/react-email/assets/110829/b09b224f-ab45-4a17-8cf5-7b7c001bf2fe)

Or you can convert the entire design into an `<Img/>` and wrap it in a `<Link>` so that you can click any part of it. A downside of this method is that you will have fixed sizing.
